src/meta/api/request/tablePaginated.ts

```diff
@@ -17,6 +17,11 @@ export type TablePaginatedDataRequestParams = BaseParams & {
   orderByDirection?: TablePaginatedOrderByDirection
 }
 
-export type TablePaginatedDataRequest = Request<never, never, never, TablePaginatedDataRequestParams>
+export type TablePaginatedDataRequest<OptionalParams extends Record<string, unknown> = never> = Request<
```

Comment:
We need to add support for optional params for TablePaginated, let me know what you think @minotogna

Reply:
Currently TablePaginated has a limited number of params
In the action itself, they are hard coded, which is fine for these params:

https://github.com/openforis/fra-platform/blob/development/src/client/store/ui/tablePaginated/actions/getData.ts#L28-L34

```
export const getData = createAsyncThunk<Returned, Props>('tablePaginated/data/get', async (props) => {
  const { assessmentName, cycleName, countryIso, orderBy, page, path, limit: _limit } = props

  const limit = _limit ?? defaultLimit

  const params: TablePaginatedDataRequestParams = {
    assessmentName,
    cycleName,
    countryIso,
    limit: String(limit),
    offset: String(page * limit),
  }
```

```
// hook calling the action:
export const useFetchData = (props: Props): void => {
  const { path, limit, counter } = props

  const dispatch = useAppDispatch()
  const { assessmentName, cycleName, countryIso } = useCountryRouteParams()
  const page = useTablePaginatedPage(path)
  const orderBy = useTablePaginatedOrderBy(path)

```

But for our new History case, we would like to include the following information:

https://github.com/openforis/fra-platform/pull/3689/files#diff-cf9cdcc4d3862f7fcf6b75f02d76638f38327cfc18a75869cff38ad56a1ece81R11-R13


```
 sectionName?: string
 targetName: string
 message: ActivityLogMessage
```

In the redux store, we have this kind of object:

```
{
  history: {
    items: {
      'disturbances-dataSources': {
        sectionLabelKey: 'description.dataSourcesPlus',
        sectionKey: 'disturbances-dataSources'
      }
    }
  }
}
```

From where we can derive sectionName and targetName but not message (`descriptionUpdate`):

`disturbances-dataSources (type: HistoryItemSectionKey)`